### DISCUSSION TITLE
Touch disturb normal function

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [rzeldent/esp32-smartdisplay#120] [Q&A]
i start testing with a brand new [esp32-3248S035C](https://www.aliexpress.com/item/1005004632953455.html).
it comes with installed demo that worked "perfectly" - including touch.

after installing your demo it end in a endless boot sequence.
Screen becomes black for millis and come back.

Terminal output:
```
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:1184
load:0x40078000,len:13232
load:0x40080400,len:3028
entry 0x400805e4
[    25][D][esp32-hal-cpu.c:244] setCpuFrequencyMhz(): PLL: 480 / 2 = 240 Mhz, APB: 80000000 Hz
[   297][I][main.cpp:26] setup(): CPU: ESP32-D0WD-V3 rev3, CPU Freq: 240 Mhz, 2 core(s)
[   298][I][main.cpp:27] setup(): Free heap: 248884 bytes
[   300][I][main.cpp:28] setup(): Free PSRAM: 0 bytes
[   305][I][main.cpp:29] setup(): SDK version: v4.4.6-dirty
....
internal DACE (916) lcd_panel.io.i2c: panel_io_i2c_rx_buffer(128): i2c transaction failed
E (917) GT911: touch_gt911_read_cfg(354): GT911 read error!
E (918) GT911: esp_lcd_touch_new_i2c_gt911(124): GT911 init failed
E (924) GT911: Error (0xffffffff)! Touch controller GT911 initialization failed!
ESP_ERROR_CHECK failed: esp_err_t 0xffffffff (ESP_FAIL) at 0x4008cbac
file: ".pio/libdeps/esp32-3248S035C/esp32_smartdisplay/src/lvgl_gt911.c" line 63
func: lvgl_touch_init
expression: esp_lcd_touch_new_i2c_gt911(io_handle, &touch_config, &touch_handle)

abort() was called at PC 0x4008cbaf on core 1

Backtrace: 0x400841b5:0x3ffb20c0 0x4008cbb9:0x3ffb20e0 0x40092b41:0x3ffb2100 0x4008cbaf:0x3ffb2180 0x400f8ce3:0x3ffb21a0 0x400f8978:0x3ffb2230 0x400d296b:0x3ffb2250 0x401235aa:0x3ffb2290

ELF file SHA256: c6a58fecf751cb5e

Rebooting...
```

When i deactived touch by changing 
`"-DBOARD_HAS_TOUCH",` 
to 
`"-DBOARD_HAS_TOUCH_no",`

the boot loop not happend anymore.

i checked the wiring and the board definition should be fine.
here is a part of the provided schematic by the seller:

![ESP32-3248S035-LCM-V1 1](https://github.com/rzeldent/esp32-smartdisplay/assets/16272493/6dad4be7-4347-4816-b946-9c163c144c48)

**Comment by rzeldent** [comment]
Are you sure you have a capacitive version? Do you have a picture of the back?
Try the develop branch...

**Comment by ChrisVonFrosta** [comment]
![Unbenannt](https://github.com/rzeldent/esp32-smartdisplay/assets/16272493/8a3033c9-2991-4672-a3fd-6b806b89c0e5)

no i am not sure. But it was offered an bought as one (capacative touch)

**Comment by rzeldent** [comment]
I do not see a GT911 chip. Suspect the U3 is an XPT2046
![image](https://github.com/rzeldent/esp32-smartdisplay/assets/7620231/6c4ce73a-c357-43c7-b080-4edaf9265205)
The XPT2046 is a resistive controller. Try if the esp32-3248S035R works for you.

**Reply by ChrisVonFrosta** [reply]
Damn, you are so right!
Thank you very much!!
![Unbenannt-1](https://github.com/rzeldent/esp32-smartdisplay/assets/16272493/49cf85be-0909-4d45-aaef-b55a65214f79)

**Reply by rzeldent** [reply]
👍 Enjoy!
